### PR TITLE
First initial commit of json conversion with test output and examples.

### PR DESCRIPTION
**Pull request by Rileyj-m**
This should be everything that we need to convert the csv to a json. There is still some things that need to be added. We need to update documentation on how they can use it in case of failure, and we also need to add some logic to handle grabbing specific values from the json. Right now, this is just cleaning up the json file. You can see what is being done by looking in the TEST folder, and there is a file in there that shows what the json looks like before any conversion. If you look inside the jsonStorage dir, you can see what it looks like after all the conversions.

### CODE REVIEW

File: csvToJson/csvToJson.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    constructor(csvFileLocation, jsonFileLocation){
+        this.csvFileLocation = csvFileLocation;
+        this.jsonFileLocation = jsonFileLocation;
+    }
+
+    // function to convert the csv file to a json object
+    convertCsvToJson(){
+        // csvHandler
+        const csvhandler = new csvHandler(this.csvFileLocation);
+        console.log("success");
```
Comment (Rileyj-m) [inline csvToJson/csvToJson.js:16]: I can remove these when time comes for final production. I think we should keep them for error checking in the future.

**Review by Maximilian741 — APPROVED** [review]
That looks very good to me after briefly looking it over.  Thank you for doing that.

File: csvToJson/csvToJson.js (hunk 1 above)
Comment (Maximilian741) [reply csvToJson/csvToJson.js:16]: I think that is a good idea.  It would definitely help.

Comment (Rileyj-m) [reply csvToJson/csvToJson.js:16]: good to know! I thought so too

**Comment by Rileyj-m** [comment]
No worries on doing this, it was part of the job ya know. I can help you use it. It's always a little confusing starting to use something that you didn't create. Once you get the functions now, it will be easy!

**Comment by Maximilian741** [comment]
Well I will definitely be asking you for help when I run into questions.  Working on getting a base website going now!

**Pull request merged**